services/common/messages/stats.proto

```diff
@@ -19,8 +19,18 @@ message PingTimes {
         uint32 ms = 5;
     }
 
+    message DevicePing {
+        string name = 1;
+        string host = 2;
+        bool online = 3;
+        uint32 ms = 4;
+    }
+
     double time = 1;
+    // Use service_pings instead, will remove.
     repeated ServicePing list = 2;
```

Comment:
are we going to actually remove this now

Reply:
Nah not right now, but the dashboard service is using it at the moment. Can drop the field after the dashboard service is retired with a Grafana dashboard.